packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -188,8 +185,8 @@ contract Migration is Script, UsingRegistry, MigrationsConstants {
    * Entry point of the script
    */
   function run() external {
-    // TODO check that this matches deployerAccount and the pK can be avoided with --unlock
-    vm.startBroadcast(deployerAccount);
+    // TODO check that this matches DEPLOYER_ACCOUNT and the pK can be avoided with --unlock
+    vm.startBroadcast(DEPLOYER_ACCOUNT);
```

Comment:
more future-proof to leave `deployerAccount` as-is whiting the code, but doing `deployerAccount= DEPLOYER_ACCOUNT` at the begining

Reply:
the scripts are able to consume the env variables of the host without entry extra step or is there anything I'm missing?

Reply:
Personally, I prefer `DEPLOYER_ACCOUNT` over `deployerAccount = DEPLOYER_ACCOUNT`, because it's clear that this is a predefined static address and not a variable.